test/unit/pusher_test.rb

```diff
@@ -287,12 +287,9 @@ class PusherTest < ActiveSupport::TestCase
       assert_not_nil @rubygem.versions.last.info_checksum
     end
 
-    should "call GemCachePurger" do
-      assert_received(GemCachePurger, :call) { |obj| obj.with(@rubygem.name).once }
-    end
-
     should "enque job for updating ES index, spec index and purging cdn" do
       assert_difference 'Delayed::Job.count', 2 do
+        @rubygem.expects(:update_attributes_from_gem_specification!).with(@cutter.version, nil)
```

Comment:
I don't think this addition is needed. Also, remove `@cutter.save` from setup and call in each `should` block. This test will fail with:
> "Delayed::Job.count" didn't change by 2.
Expected: 2
  Actual: 5

Delayed job count change by 5 is real/correct assertion. gem indexer, es indexing, and 3 more jobs for purging gem.name, versions and info/gem.name cache.

Reply:
Also, please add explanation for change in job count to commit message.